value.go

```diff
@@ -168,11 +180,11 @@ type LTable struct {
 	k2i     map[LValue]int
 }
 
-func (tb *LTable) String() string                     { return fmt.Sprintf("table: %p", tb) }
-func (tb *LTable) Type() LValueType                   { return LTTable }
-func (tb *LTable) assertFloat64() (float64, bool)     { return 0, false }
-func (tb *LTable) assertString() (string, bool)       { return "", false }
-func (tb *LTable) assertFunction() (*LFunction, bool) { return nil, false }
+func (tb LTable) String() string                     { return fmt.Sprintf("table: %p", tb) }
```

Comment:
What made you decide to copy `LTable` for all these method calls? I don't think removing the pointer receiver here is a good idea.